Infrastructure/Identity/UserService.cs

```diff
@@ -56,4 +56,10 @@ private async Task<bool> TenantExistsAsync(int tenantId)
     {
         return await _dbContext.Tenants.AnyAsync(x => x.Id == tenantId);
     }
+
+    public async Task<bool> CanManageOrdersAsync(Employee employee)
+    {
+        var authorized = employee.Position >= PositionType.Cashier;
+        return await Task.FromResult(authorized);
+    }
```

Comment:
budamas parduotuves A cashier galesiu perziuret parduotuves B cashier, nes nepaminetas tenant

Reply:
+ nera includinti manageriai ir adminai, tai nei vienas is ju negales perziuret orderiu